scripts/xtensa-build-zephyr.py

```diff
@@ -1033,6 +1106,8 @@ class InstFile:
 		gzip_res.result() # throws exception if gzip unexpectedly failed
 	gzip_threads.shutdown()
 
+	if args.deployable_build and platform_configs[platform].ipc4:
+		install_lib(platform, sof_output_dir, abs_build_dir)
```

Comment:
Move this before `sof-info/` so the code does not ping-pong between the two. Who knows, we may even want to checksum modules in the future.